src/napari_matplotlib/base.py

```diff
@@ -133,6 +124,30 @@ def draw(self) -> None:
         This is a no-op, and is intended for derived classes to override.
         """
 
+    def apply_napari_colorscheme(self):
+        """
+        Apply napari-compatible colorscheme to the axes object.
+        """
+        if self.axes is None:
+            return 0
+                # changing color of axes background to napari main window color
+        self.canvas.figure.patch.set_facecolor("#262930")
```

Comment:
Could you stick the napari window color in a variable so it's easier to re-use (something like `NAPARI_WINDOW_COLOR = "#262930"`

Reply:
Fixed in e9f4345ad3e874d8489eb38efc25182f882fde13